.github/actions/build-dist/action.yml

```diff
@@ -20,7 +20,6 @@ runs:
 
     - name: Install
       shell: bash
-      if: steps.cache-node-modules.outputs.cache-hit != 'true'
```

Comment:
:crying_cat_face: caching `node_modules` (provided `.eslint` and `.prettier` don't muck about in there, as they do by default for some reason) has been really stable, and saves an appreciable real amount of time...  meanwhile, without a way to clean out the yarn cache, it just gets bigger, and misses a lot.

it _does_ opens us up for e.g. the `svgo` issues, which seem to happen _anyway_ when the runners change their `apt` deps, i guess.

Reply:
I don't think this one was used anyway, the `jupyterlab/maintainer-tools/.github/actions/base-setup` action above should already handle caching.

Reply:
For `svgo` maybe we should remove it from the CI / dodo tasks. I don't know if we need to optimize the images all the time, this could be done occasionally when needed. And also setting up a fresh new env almost always leaves the git repo in a dirty state with some icon files being modified.

Reply:
>  should already handle caching.

... of the ever-expanding, never-pruned `~/.cache/yarn`, which then _still_ requires compiling binaries from source, sometimes, and invoking the solver.

> for `svgo`

My point was "_any_ crazy `node_modules`," but `svgo` in specific _could_ be replaced with the stable, but slightly less efficient pure-python `scour`. But maybe more closed-form, and would cause less thrash.

Anything that "only gets run sometimes,".... doesn't.

Reply:
Anyway that's off topic, I opened https://github.com/jupyterlite/jupyterlite/issues/1004 to discuss this more.